validation-test/Sema/type_checker_crashers_fixed/rdar103739206.swift

```diff
@@ -0,0 +1,20 @@
+// RUN: not %target-swift-frontend %s -typecheck
+
+protocol RawTokenKindSubset {}
+
+struct Parser {
+  func canRecoverTo<Subset: RawTokenKindSubset>(anyIn subset: Subset.Type) {
+    if let (kind, handle) = self.at(anyIn: subset) {
+    }
+  }
+
+  func at(_ keyword: Int) -> Bool {}
+
+  func at(
+<<<<<<< HEAD
```

Comment:
Is the git markers part of the reproducer? (Looks manually inserted) 
If that is the case, do you think this is worth a comment just so someone in the future attempts to "fix it"?

Reply:
Yeah, it’s part of the issue, I will outline that in the message.

Reply:
✅